dashboards/kubernetes/node.json

```diff
@@ -0,0 +1,300 @@
+{
+  "metric_keys": [
+    "node_cpu_usage_nano_cores"
+  ],
+  "dashboard": {
+    "title": "Kubernetes Nodes",
+    "description": "",
+    "visuals": [
+      {
+        "title": "Node CPU Usage",
+        "description": "node_cpu_usage_nano_cores",
```

Comment:
These descriptions don't explain anything to me. Let's remove them if it's just for testing.

Reply:
These are the internal metric names, so they'll probably make sense to Kubernetes users.

Reply:
I can't the names of these metrics anywhere, so I wouldn't be so sure. The same way I can't find that you're using the kubernetes metric names, [like you say here](https://github.com/appsignal/public_config/pull/64#discussion_r1555772715). In the library I see it being mapped from a JSON struct that doesn't use the same naming: https://github.com/appsignal/appsignal-kubernetes/blob/0b3f39d65ba99622ab3e647e8e4c012ee944baca/src/main.rs#L97-L167

Do you have any links to docs or source code that mentions these metric names?